extensions/algolia/fieldgroups/algolia-profile.schema.json

```
@@ -0,0 +1,85 @@
+{
+  "$id": "https://ns.algolia.com/xdm/algolia/fieldgroup/profile",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Algolia Profile Details",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:status": "experimental",
+  "meta:license": [
+    "Copyright 2024 Algolia Inc. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "description": "Algolia gathers user's click/conversion events to build user profiles for personalization and A/B testing. This field group properties aligns with the Algolia user profile structure.",
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "definitions": {
+    "algolia-profile": {
```

```suggestion
    "algolia-profile": {
        "type": "object",
        "properties": {
          "algoliaProfile": {
          "type": "object",
```
We need to add a namespace for algolia profile fields so that they won't collide with other experience event fields. To add a namespace, please wrap the entire schema definition inside an object "algoliaProfile"

---

I have updated the schema.